src/main/webapp/availabilityTable.jsp

```diff
@@ -0,0 +1,24 @@
+<%@ page import="com.google.sps.data.AvailabilityTimeSlots" %>
+<%
+  AvailabilityTimeSlots list = new AvailabilityTimeSlots(request.getParameter("timeZoneOffset"));
+  pageContext.setAttribute("list", list);
+%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %> 
+
+<table class="table table-sm text-center">
+  <thead>
+    <tr>
+      <th scope="col">${list.getTimeSlots().get(0).getDate()}</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- TODO: Allow clicking and scrolling over multiple slots to select them.-->
+    <c:forEach items = "${pageScope.list.getTimeSlots()}" var = "timeSlot">
+      <tr>
+        <td onclick="switchTile(this)" data-utc="${timeSlot.getUTCEncoding()}" class="${timeSlot.getClassName()}">
```

Comment:
What is "data-utc" doing here? I see no crossreferences in this PR.

Reply:
This is what will be used when the submit button is clicked. First we will filter through, grabbing only the slots that are currently selected, and then their utc encodings will be sent to the backend to be processed and update datastore.

Reply:
Perhaps add this when you need it?

"Will be used" is the harbinger for "what was this supposed to be for and why is it unused" ten years later.

Reply:
Added a meaningful TODO near Update Availability button.